.github/workflows/deploy-auth.yml

```
@@ -2,7 +2,7 @@ name: Deploy Auth Service
 
 on:
   push:
-    branches: [develop]
+    branches: [develop,"feature/**"]
```

YAMLlint (1.35.1)

[error] too few spaces after comma

(commas)